certbot/CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Certbot adheres to [Semantic Versioning](https://semver.org/).
   it caused a previously-existing inability to parse empty files to become more
   visible. We have now added the ability to correctly parse empty files, so that
   message should only show for more significant errors.
+* Fixed parsing of `Define`d values the apache component to allow for `=` in the value.
```

Comment:
This looks like it works perfectly, thank you very much.

Just a couple of things with the changelog:

1. Certbot 1.18.0 has been released since this PR was created, please merge updated `master` into the PR and move this changelog entry up to the section for 1.18.0.
2. I propose clarifying the changelog entry slightly:

```suggestion
* Fixed parsing of `Define`d values in the Apache plugin to allow for `=` in the value.
```